packages/components/src/shared/model.ts

```diff
@@ -15,6 +15,13 @@ export type GlobalTextProps = {
 	pattern?: string;
 };
 
+export type ImageProps = {
+	imgAlt?: string;
+	imgSrc?: string;
+	imgHeight?: number | string;
+	imgWidth?: number | string;
```

Comment:
```suggestion
	imgHeight?: number;
	imgWidth?: number;
```

> The intrinsic width of the image in pixels. Must be an integer without a unit.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-width

Reply:
how is this resolved ?